### PR TITLE
Add Spanish translation

### PR DESCRIPTION
**Pull request by andreynovikov95**
Fix the language line in README.rst for [PR](https://github.com/adrienverge/PhotoCollage/pull/61)

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -37,7 +37,7 @@ Features:
 * save high-resolution image
 * works even with a large number of photos (> 100)
 * integrates into the GNOME environment
-* available in English, French, German, Czech, Italian, Bulgarian, Dutch and Russian
+* available in English, Bulgarian, Czech, Dutch, French, German, Italian, Russian and Spanish
```
Comment (adrienverge) [inline README.rst:40]: Can you keep the same order and just append Spanish?

File: po/ru.po [hunk 2]
```diff
@@ -5,18 +5,18 @@
 #
 msgid ""
 msgstr ""
-"Project-Id-Version: PhotoCollage 1.4.3\n"
+"Project-Id-Version: PhotoCollage 1.4.4\n"
```
Comment (adrienverge) [inline po/ru.po:8]: No need to update `ru.po` in this PR, can you exclude it from this PR?

**Review by adrienverge — COMMENTED** [review]
@andreynovikov95 thanks :+1: Please see my 2 comments.

File: README.rst (hunk 1 above)
Comment (andreynovikov95) [reply README.rst:40]: Ok. But when there are more and more supported languages, it is easier to find own in the list, when it sorted in alphabetical order.

File: po/ru.po (hunk 2 above)
Comment (andreynovikov95) [reply po/ru.po:8]: Ok

File: README.rst (hunk 1 above)
Comment (adrienverge) [reply README.rst:40]: Sure, but I think we'll stop adding languages here at some point.

Comment (andreynovikov95) [reply README.rst]: When will the new version be released in distributions like Ubuntu, Linux Mint and other? How long is this process?

Comment (andreynovikov95) [reply README.rst]: @adrienverge, I can't run the program with the Russian language according to the "Hacking" instructions. Always runs with English. Is the instruction up-to-date?

Comment (adrienverge) [reply README.rst]: > When will the new version be released in distributions like Ubuntu, Linux Mint and other? How long is this process?

I don't know, I don't maintain these distributions packages. I can make a new release in the GitHub repo, if needed.

> I can't run the program with the Russian language according to the "Hacking" instructions. Always runs with English. Is the instruction up-to-date?

I don't know. You'll have to debug.

**Comment by adrienverge** [comment]
> No need to update `ru.po` in this PR, can you exclude it from this PR?

It looks like this comment is still not addressed. Please squash all commits into one, push, and warn me when the PR 100% ready. Thanks!

File: README.rst (hunk 1 above)
Comment (andreynovikov95) [reply README.rst]: The new release would be great!

**Comment by andreynovikov95** [comment]
@adrienverge, can you to check it again?

**Review by adrienverge — APPROVED** [review]
Thanks!

File: README.rst (hunk 1 above)
Comment (adrienverge) [reply README.rst]: > The new release would be great!


Done: version 1.4.5 released :+1:

**Pull request merged**